content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -136,7 +136,7 @@ different Kubernetes components.
 | `LocalStorageCapacityIsolation` | `false` | Alpha | 1.7 | 1.9 |
 | `LocalStorageCapacityIsolation` | `true` | Beta | 1.10 | |
 | `LocalStorageCapacityIsolationFSQuotaMonitoring` | `false` | Alpha | 1.15 | |
-| `LogarithmicScaleDown` | `false` | Alpha | 1.21 | |
+| `LogarithmicScaleDown` | `true` | Beta | 1.22 | |
```

Comment:
Don't change it, but add a 2nd row for Beta version like above.

Reply:
Updated